airflow/utils/helpers.py

```diff
@@ -23,7 +23,12 @@
 
 from datetime import datetime
 from functools import reduce
-from collections import Iterable
+try:
+    # Fix Python > 3.7 deprecation
+    from collections.abc import Iterable
+except ImportError:
+    # Preserve Python < 3.3 compatibility
```

Comment:
The mimimum Python version we support is now 3.5 so I don't think we need this. (unless we also want to backport this to 1.10.x, where we still support py2.7)

Reply:
Thanks, it's also present in 1.10.x branch. The fix was based on https://issues.apache.org/jira/browse/AIRFLOW-3009 where similar problem was fixed in master and also seemed to be backported to 1.10.3.